unicornia/src/Blocks/custom/carousel/carousel.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Template for the Carousel Block.
+ *
+ * @package Unicornia
+ */
+
+use UnicorniaVendor\EightshiftLibs\Helpers\Components;
+use Unicornia\Manifest\Manifest;
+
+$manifest = Components::getManifest(__DIR__);
+
+$blockClass = Components::checkAttr('blockClass', $attributes, $manifest);
+$blockJsClass = Components::checkAttr('blockJsClass', $attributes, $manifest);
+$isLoop = Components::checkAttr('isLoop', $attributes, $manifest);
+$showItems = Components::checkAttr('showItems', $attributes, $manifest);
+
+$carouselClass = Components::classnames([
+	$blockClass,
+	$blockJsClass,
+	'swiper-container',
+]);
+
+?>
+
+<div
+	class="<?php echo esc_attr($carouselClass); ?>"
+	data-swiper-loop="<?php echo esc_attr($isLoop); ?>"
+	data-show-items="<?php echo esc_attr($showItems); ?>">
+
+	<div class="<?php echo esc_attr('swiper-wrapper'); ?>">
+		<?php
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		echo $innerBlockContent;
+		?>
+	</div>
+
+	<div class="<?php echo esc_attr($blockClass); ?>__nav">
+		<div class="<?php echo esc_attr($blockJsClass); ?>-prev-arrow <?php echo esc_attr($blockClass); ?>__arrow-prev">
+			<img src="<?php echo esc_attr(apply_filters(Manifest::MANIFEST_ITEM, 'arrow-left.svg')); ?>"
+			alt="<?php echo esc_attr(__('Previous', 'unicornia')); ?>">
```

Comment:
```suggestion
			alt="<?php esc_attr_e('Previous', 'unicornia'); ?>">
```

https://developer.wordpress.org/reference/functions/esc_attr_e/

WordPress has tons of useful shortcuts 😄 

The text domain is set so that's great 👍🏼